test/recipes/default/user_test.rb

```diff
@@ -0,0 +1,71 @@
+describe group('stanley') do
+  it { should exist }
+end
+
+describe user('stanley') do
+  it { should exist }
+  its('group') { should eq 'stanley' }
+  its('home') { should eq '/home/stanley' }
+end
+
+describe passwd.users(/stanley/) do
+  its('homes') { should eq ['/home/stanley'] }
+  its('shells') { should eq ['/bin/bash'] }
```

Comment:
Well I went a little bit overboard with that InSpec resource. `/bin/bash` is the shell that gets assigned to the user when the resource is managed by Chef. If that is not a requirement for StackStorm (maybe for the local action runner?) we can get rid of it.